services/account-without-juggler/repositories/account/datasources/mysqlconn.ts

```diff
@@ -0,0 +1,229 @@
+const debug = require("debug")("accountsansjuggler");
+const mysql = require("mysql");
+import {
+  CrudConnector,
+  EntityData
+} from "loopback-next/packages/repository/src/crud-connector";
+import {
+  Class,
+  DataSource,
+  Entity,
+  Filter,
+  ObjectType,
+  Options,
+  Where
+} from "loopback-next/packages/repository";
+
+export class MySqlConn implements CrudConnector {
+  private connection: any;
+
+  constructor(config: Object) {
+    this.connection = mysql.createConnection(config);
+    if (this.connection.state !== "authenticated") {
+      this.connect().then(
+        function(res) {
+          debug("Connection success ", res);
+        },
+        function(err) {
+          debug("err connecting", err);
+        }
+      );
+    }
+  }
+  name: "mysql";
+  interfaces?: string[];
+  connect() {
+    let self = this;
+    return new Promise<void>(function(resolve, reject) {
+      self.connection.connect(function(err: any) {
+        if (err !== null) return reject(err);
+        resolve(err);
+      });
+    });
+  }
+  disconnect() {
+    let self = this;
+    return new Promise<void>(function(resolve, reject) {
+      self.connection.end(function(err: any) {
+        if (err !== null) return reject(err);
+        resolve(err);
+      });
+    });
+  }
+  ping() {
+    let self = this;
+    return new Promise<void>(function(resolve, reject) {
+      self.connection.ping(function(err: any) {
+        if (err !== null) return reject(err);
+        resolve(err);
+      });
+    });
+  }
+
+  public updateAll(
+    modelClass: Class<Entity>,
+    data: EntityData,
+    where: Where,
+    options: Options
+  ): Promise<number> {
+    throw new Error("Not implemented yet.");
+  }
+
+  public create(
+    modelClass: Class<Entity>,
+    entity: EntityData,
+    options: Options
+  ): Promise<EntityData> {
+    let self = this;
+    let sqlStmt = "INSERT INTO " + modelClass.name + " VALUES ";
+    var vals = [];
+    for (var prop in entity) {
+      vals.push(entity[prop]);
+    }
+    sqlStmt += "(" + vals.join(",") + ")" + ";";
+    debug("insert stmt ", sqlStmt);
+    return new Promise<Account>(function(resolve, reject) {
+      self.connection.query(sqlStmt, function(err: any, count: any) {
+        if (err !== null) return reject(err);
+        resolve(count.affectedRows);
+      });
+    });
+  }
+
+  public save(
+    modelClass: Class<Entity>,
+    entity: EntityData,
+    options: Options
+  ): Promise<EntityData> {
+    throw new Error("Not implemented yet.");
+  }
+
+  public find(
+    modelClass: Class<Entity>,
+    filter: Filter,
+    options: Options
+  ): Promise<EntityData[]> {
+    let self = this;
+    let sqlStmt = "SELECT * FROM " + modelClass.name;
+    if (filter.where) {
+      let sql = "?? = ?";
+      let formattedSql = "";
+      for (var key in filter.where) {
+        formattedSql = mysql.format(sql, [key, filter.where[key]]);
+      }
+      sqlStmt += " WHERE " + formattedSql;
```

Comment:
FWIW, I think this is suboptimal performance wise. Each query will be a unique string, therefore the SQL engine has very little opportunities for caching pre-computed values.

Ideally, the SQL statement should looks like this:

```
SELECT * FROM model WHERE foo = ? AND bar = ?
```

The same comment applies to all other places where you are creating custom SQL statements.